modelforge/tests/data/config.toml

```diff
@@ -33,6 +33,12 @@ dataset_name = "QM9"
 version_select = "nc_1000_v0"
 num_workers = 4
 pin_memory = true
+properties_of_interest = ["atomic_numbers", "geometry", "internal_energy_at_0K", "dipole_moment"]
```

Comment:
We may need documentation for available options and possibly corresponding types and shapes (e.g., a scalar, an array with a size equal to the number of atoms).

Reply:
this is a good point.  We need to have in the documentation and wiki summarizes of each dataset and what the available properties of interest are.  I can work on that in a separate PR (likely best to set up a script to autogenerate this .rst file)